packages/nocms-plugin-resources-json/package.json

```diff
@@ -6,7 +6,7 @@
 	"scripts": {
 		"cover": "istanbul cover ./node_modules/mocha/bin/_mocha",
 		"coveralls": "npm run cover -- --report lcovonly && cat ./coverage/lcov.info | coveralls",
-		"test": "mocha"
+		"test": "mocha --opts /test/mocha.opts"
```

Comment:
Should mocha not default to this?

Reply:
It does, it's just easy to overlook, so I'm making it more explicit if somebody should not know the mocha default config path by heart ❤️